### ISSUE TITLE
[Bug] The app does not start, crash after build

### ISSUE THREAD

**Issue by mattiabonzi** [creativetimofficial/soft-ui-react-native#32]
### Version
1.1.1
### Reproduction link
https://github.com/creativetimofficial/soft-ui-react-native
### Operating System
 Android 13 | Mac Os 14.3.1
### Device
Redmi Note 12 | Macbook Pro 2021 M1 Pro
### Browser & Version
Does not apply, I'm running it on physical device
### Steps to reproduce
1) Download the template from Creative Tim website.
2) Open a terminal and enter into the template forlder (root folder).
3) Run `npm install`
4) Run `npm start`
5) Press "a" to open the app on the connected Android device (the device is connected correctly, and all permission has been granted)
### What is expected?
The app starts and run as described in the documentation.
### What is actually happening?
The app does not start correctly and crash as soon the building process finish.

---
### Additional comments
No code as been modified, no file has been moved, just downloaded and tried to run as per documentation.

```
soft-ui-pro-react-native % npm start

> start
> ANDROID_HOME=/opt/homebrew/Caskroom/android-sdk/4333796 expo start

Starting project at /Users/mattiabonzi/Progetti/farmshop/soft-ui-pro-react-native
Starting Metro Bundler
▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
█ ▄▄▄▄▄ █ ▀▀▄ ▄██ █ ▄▄▄▄▄ █
█ █   █ ███ ▄▄ █▄ █ █   █ █
█ █▄▄▄█ █ ▄▄ █▄██▄█ █▄▄▄█ █
█▄▄▄▄▄▄▄█ █ ▀ █▄▀ █▄▄▄▄▄▄▄█
█▄▄ █▀ ▄█▄  ██▀ ▀▄▄ ▄▄██  █
██▀ █▀▀▄ █▀█▄██▄▄██▀▄█▄▄█▄█
█▄▀ ██▀▄█ ▄█▄ █ ▀█  ███▀ ▀█
█▄▄▄█▀█▄▄█▀▄▀ ▄    ▄█ ▄█▀▄█
█▄▄█▄▄▄▄▄ ▄▀█▄▀ █ ▄▄▄  ▀█ █
█ ▄▄▄▄▄ █▀▀█▄▄▄ █ █▄█ ██▀▄█
█ █   █ ██▄▄ ██ ▄▄▄▄   ▀ ██
█ █▄▄▄█ █ ▄ ▄▄▄▄ ▀▀█ ▄██▄▄█
█▄▄▄▄▄▄▄█▄▄█▄██▄▄▄▄█▄▄███▄█

› Metro waiting on exp://192.168.9.110:19000
› Scan the QR code above with Expo Go (Android) or the Camera app (iOS)

› Press a │ open Android
› Press i │ open iOS simulator
› Press w │ open web

› Press j │ open debugger
› Press r │ reload app
› Press m │ toggle menu

› Press ? │ show all commands

Logs for your project will appear below. Press Ctrl+C to exit.
› Opening on Android...
› Opening exp://192.168.9.110:19000 on 23021RAA2Y
› Press ? │ show all commands
Android Bundling complete 1804ms
 ERROR  Invariant Violation: Tried to register two views with the same name RNCMaskedView, js engine: hermes
 ERROR  Invariant Violation: "main" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes
```

**Comment by sachin8606** [comment]
I am also facing this issue. @mattiabonzi did u get any solution?

**Comment by softwareshinobi** [comment]
ughh... running into this too.

i'm trying to avoid digging my sleeves into this template to fix it.

i might have to.

there aren't that many online.

ugh.

**Comment by OrdenWills** [comment]
@softwareshinobi any progress? am stuck with it too for someone who's new to react native debugging a full app will probably take longer than writing from scratch

**Comment by seppestaes** [comment]
Same, can't open with Expo Go "Something went wrong. Sorry about that..."

**Comment by erickwarui28** [comment]
deleting node_modules/@react-native-community/masked-view folder worked for me since it's a duplicate of node_modules/@react-native-masked-view/masked-view
